gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerKotlinSupportPlugin.kt

```diff
@@ -68,4 +86,15 @@ class ComposeCompilerKotlinSupportPlugin : KotlinCompilerPluginSupportPlugin {
 
     private fun options(vararg options: Pair<String, String>): List<SubpluginOption> =
         options.map { SubpluginOption(it.first, it.second) }
-}
\ No newline at end of file
+}
+
+private const val COMPOSE_COMPILER_COMPATIBILITY_LINK =
+    "https://github.com/JetBrains/compose-jb/blob/master/VERSIONING.md"
+
+private val WARN_ABOUT_JC_COMPILER = """
+    | WARNING: You are using the 'androidx.compose.compiler' plugin in your Kotlin multiplatform project.
+    | This plugin is only guaranteed to work with JVM targets (desktop or Android).
+    | The usage with Kotlin/JS or Kotlin/Native targets is not supported and might cause issues.
+    | Make sure you are using compatible versions of the Jetpack Compose Compiler and Kotlin.
```

Comment:
We can't make compatibility table between Jetpack Compose Compiler, only between JetBrains Compose Compiler

Reply:
Thanks! it's a typo